src/docs/get-started/codelab.md

```diff
@@ -427,104 +437,96 @@ lazily, on demand.
 
  3. Add a `_buildRow()` function to `RandomWordsState`:
 
     <!-- skip -->
+    <?code-excerpt "4-infinite-list/lib/main.dart (_buildRow)" indent-by="2"?>
     {% prettify dart %}
-      class RandomWordsState extends State<RandomWords> {
-        ...
-
-        [[highlight]]Widget _buildRow(WordPair pair) {[[/highlight]]
-          [[highlight]]return ListTile([[/highlight]]
-            [[highlight]]title: Text([[/highlight]]
-              [[highlight]]pair.asPascalCase,[[/highlight]]
-              [[highlight]]style: _biggerFont,[[/highlight]]
-            [[highlight]]),[[/highlight]]
-          [[highlight]]);[[/highlight]]
-        [[highlight]]}[[/highlight]]
+      Widget _buildRow(WordPair pair) {
+        return ListTile(
+          title: Text(
+            pair.asPascalCase,
+            style: _biggerFont,
+          ),
+        );
       }
     {% endprettify %}
 
- 4. Update the `build` method for RandomWordsState to use
+ 4. Update the `build()` method for `RandomWordsState` to use
     `_buildSuggestions()`, rather than directly calling the word
     generation library.
     ([Scaffold](https://docs.flutter.io/flutter/material/Scaffold-class.html)
     implements the basic Material Design visual layout.)
 
+    Replace the method with the following code:
+
     <!-- skip -->
+    <?code-excerpt "4-infinite-list/lib/main.dart (RWS-build)" indent-by="2"?>
     {% prettify dart %}
-      class RandomWordsState extends State<RandomWords> {
-        ...
-        @override
-        Widget build(BuildContext context) {
-          [[strike]]final wordPair = WordPair.random();[[/strike]] // Delete these two lines.
-          [[strike]]return Text(wordPair.asPascalCase);[[/strike]]
-          [[highlight]]return Scaffold ([[/highlight]]
-            [[highlight]]appBar: AppBar([[/highlight]]
-              [[highlight]]title: Text('Startup Name Generator'),[[/highlight]]
-            [[highlight]]),[[/highlight]]
-            [[highlight]]body: _buildSuggestions(),[[/highlight]]
-          [[highlight]]);[[/highlight]]
-        }
-        ...
+      @override
+      Widget build(BuildContext context) {
+        return Scaffold(
+          appBar: AppBar(
+            title: Text('Startup Name Generator'),
+          ),
+          body: _buildSuggestions(),
+        );
       }
     {% endprettify %}
 
- 5. Update the build method for MyApp, changing the title,
+ 5. Update the `build()` method for `MyApp`, changing the title,
     and changing the home to be a RandomWords widget.
 
-    Replace the original method with the highlighted build method below:
+    Replace the original method with the highlighted method below:
 
     <!-- skip -->
+    <?code-excerpt "4-infinite-list/lib/main.dart (MyApp)" replace="/(\n  )(.*)/$1[[highlight]]$2[[\/highlight]]/g" indent-by="2"?>
     {% prettify dart %}
       class MyApp extends StatelessWidget {
-        @override
+        [[highlight]]@override[[/highlight]]
```

Comment:
The highlights here are inconsistent with the (lack of) highlights above. If the build() method already existed in this class (did it?), then highlighting only the method contents (the stuff that's changed) seems right to me.

Reply:
This will also be changed to a pretty diff soon.